Marlin/src/module/planner.h

```diff
@@ -827,9 +838,11 @@ class Planner {
       static void autotemp_update();
     #endif
 
+    #define JUNC_SQ(N,ST) (junction_deviation_mm * (N) * (ST) / (1.0f - (ST)))
+
     #if HAS_LINEAR_E_JERK
       FORCE_INLINE static void recalculate_max_e_jerk() {
-        #define GET_MAX_E_JERK(N) SQRT(SQRT(0.5) * junction_deviation_mm * (N) * RECIPROCAL(1.0 - SQRT(0.5)))
+        #define GET_MAX_E_JERK(N) SQRT(JUNC_SQ(N,SQRT(0.5)))
         #if ENABLED(DISTINCT_E_FACTORS)
```

Comment:
@thinkyhead 
The simplification with omitting the `RECIPROCAL()` macro is correct - ST is not near zero in the here used case. But in the definition of JUNC_SQ(N,ST) is not. Here ST is a variable that could be 1.0f resulting in a division by zero. 
As long as the code is used only once - better don't make a (faulty) macro from it.

Reply:
It's a reasonable point. I wanted to highlight for the reader the relationship between the two points in the code that apply the formula. In neither case is division by zero a danger, but if misapplied, it would be bad.